tests/suites.c

```diff
@@ -298,7 +299,6 @@ static int execute_test_case(int svr_argc, char** svr_argv,
         if (cliArgs.argc + 2 > MAX_ARGS)
             printf("cannot add the magic port number flag to client\n");
         else {
-            char portNumber[8];
```

Comment:
fsanitizer reported use of variable declared in local scope after scope had ended.